mmv1/third_party/terraform/website/docs/r/storage_bucket.html.markdown

```diff
@@ -69,6 +69,28 @@ resource "google_storage_bucket" "auto-expire" {
 }
 ```
 
+## Example Usage - Life cycle settings for storage bucket objects with `no_age` enabled
+When creating a life cycle condition that does not also include an `age` field, it's important to include the `no_age` flag so your new bucket does not unintentionally create an extreanous condition that may interact incorrectly with your other options.
```

Comment:
```suggestion
When creating a lifecycle rule that does not include an `age` setting, a default `age` of `0` is set. Set `no_age` to `true` to avoid this. 
```

Reply:
nit: Let's be more direct about why!

Reply:
Wasn't sure how direct about the situation we wanted to be, added in much more direct details!